config/initializers/datadog.rb

```diff
@@ -1,6 +1,10 @@
 if ENV['DATADOG_RAILS_APM']
+  require 'dalli'
+  require 'ddtrace'
```

Comment:
@Matt-Yorkley Am I right that, based on the `Gemfile`, these two requires are not actually needed?